app/common/auth/authentication-events.run.js

```diff
@@ -38,7 +37,8 @@ function AuthenticationEvents($rootScope, $location, Authentication, Session, _,
                 if (redirect) {
                     $location.url(redirect);
                 }
-                noReload || $route.reload();
+                noReload || $state.go($state.$current, null, { reload: true }); // in favor of $route.reload();
```

Comment:
I think there is a `$state.reload()` method https://ui-router.github.io/ng1/docs/latest/classes/state.stateservice.html#reload